ArcGISRuntimeSDKQt_QMLSamples/Maps/MinMaxScale/MinMaxScaleSample.json

```diff
@@ -0,0 +1,5 @@
+{
+    "source" :  "MinMaxScaleSample.qml",
```

Comment:
MinMaxScaleSample.qml should change to MinMaxScale.qml. You will need to rename the files as well